ecs_fargate/datadog_checks/ecs_fargate/ecs_fargate.py

```diff
@@ -67,6 +67,12 @@ def c_is_excluded(name, image, namespace=""):
 ]
 MEMORY_RATE_METRICS = ['pgpgin', 'pgpgout', 'pgmajfault', 'pgfault']
 IO_METRICS = {'io_service_bytes_recursive': 'ecs.fargate.io.bytes.', 'io_serviced_recursive': 'ecs.fargate.io.ops.'}
+STORAGE_STATS_METRICS = {
+    'read_count_normalized': 'ecs.fargate.io.read.count',
```

Comment:
I think we should add a comment here that mentions that this is Windows specific. Same for IO_METRICS above indicating that it is Linux specific.

Shouldn't these metrics have the same name as the Linux ones? I'm not sure if they're equivalent.
For example, instead of defining `ecs.fargate.io.read.size` we could use the existing `ecs.fargate.io.bytes.read`.

`metadata.csv` should be updated accordingly.

Reply:
They seem to be equivalent, so we could reuse Linux metrics names. I'll do the modifications 👍